### ISSUE TITLE
Firefox ESR 38.0.3. bild.de shows paywall, Firefox 41.0.2. not

### ISSUE THREAD

**Issue by synthor** [gorhill/uBlock#872]
Hi,

I want to deploy Firefox 38.0.3. ESR with uBlock 1.3.1 and a preconfigured Sqlite-Databasefile in my company (All machines are Windows 7 x64 with latest updates installed).

Deploying works as expected, but with the ESR version uBlock doesn't seem to work properly.

e.g. bild.de (the famous newspaper site, which installed a paywall a few weeks before) with ESR 38.0.3. uBlock doesn't filter correct and I get the paywall.

I just modified one line in the .xpi to place the Sqlite-Database outside of the userprofile (that would lead to a feature request: a possibility to change the default path extension-data/uBlock0.sqlite would be nice, but I will look to make a separate PR for that) and thought that this and a filesystem permission issue would cause this behaviour).

But nope, even when I use the original 1.3.1. xpi and create a new config in the users profile bild.de won't load. I can see that sacdn.com and smartadservers.com are blocked by uBlock but the paywall still appears?!

I thought it's a bug in uBlock and also tried 1.3.2. rc4 yesterday...but nope.

Now, with installed 41.0.2. instead of ESR 38.0.3., my modified xpi and the preconfigured Database bild.de loads like it should.

What can I do to help you with that? I can provide logfiles or something...otherwise just try to reproduce with ESR 38.0.3 and 1.3.1 or newer.

Edit: Additional settings I've made

- Experienced user
- Block IP WebRTC
- All standard filters + Adblock warning remov. list, anti-adblockkiller reek, easyprivacy, fanboy's enhanced tracking list, fanboy's annoyance list, fanboy's social blocking list, fanboy+easylist merged ultimate list, eazylist germany

Regards
Thorsten

**Comment by gorhill** [comment]
The [doc regarding inline script tag filtering](https://github.com/gorhill/uBlock/wiki/Inline-script-tag-filtering#caveats):

> **Caveats** ... Not supported on older versions of Firefox

Workaround for those browsers not supporting inline script tag filtering is to filter inline script wholesale, using a dynamic filtering rule or a static filter. This can however breaks some functionality on the site.

**Comment by synthor** [comment]
Thanks for your info.

I started reading the wiki yesterday (dynamic filtering) and will easily create a custom filter for this one site. As I know in my mind, you'd just have to block those two javascripts and the paywallcode doesn't get executed.

**Comment by synthor** [comment]
It is really an inline script which shows the paywall. I only have read about those two third-party servers. 

Then the employees still won't be able to read bild.de until FF 45 ESR is out. My boss will like that decison. :+1:

**Comment by Serkan-devel** [comment]
But who wants to get their news from bild anyways?
